Makefile

```diff
@@ -1,31 +1,65 @@
 .PHONY: all test
 
-BUILD_CONTAINER_NAME=calico/libcalico_test_container
-BUILD_CONTAINER_MARKER=libcalico_test_container.created
+GLIDE_CONTAINER_NAME?=dockerepo/glide
+TEST_CONTAINER_NAME=calico/libcalico_test_container
+TEST_CONTAINER_MARKER=libcalico_test_container.created
+VENDOR_CREATED=vendor/.vendor.created
 
 GO_FILES:=$(shell find lib -name '*.go')
 
 default: all
 all: test
 test: ut
 
+.PHONY: vendor
```

Comment:
Kinda seems wierd to have vendor be PHONY, given it's actually a real directory that we want to build.  What's the reason for this?